site/using.md

```diff
@@ -20,46 +20,28 @@ You need to tell `fluxctl` where to find the Flux API and to make
 the pod accessible to the command-line client `fluxctl`, you
 will need to expose the API outside the cluster.
 
-A simple way to do this is to piggy-back on `kubectl port-forward`,
-assuming you can access the Kubernetes API:
+By default, fluxctl will attempt to forward to your Flux instance in the `default`
+namespace. You can also specify the namespace that your Flux instance lives in with the
+`-f` or `--k8s-forward-namespace` flag:
 
-```sh
-fluxpod=$(kubectl get pod -l name=flux -o name | awk -F / '{ print $2; }')
-kubectl port-forward "$fluxpod" 10080:3030 &
-export FLUX_URL="http://localhost:10080/api/flux"
 ```
-
-Exporting `FLUX_URL` is enough for `fluxctl` to know how to contact
-the daemon. You could alternatively supply the `--url` argument
-each time.
-
-## Flux API service
-
-Now you can easily query the Flux API:
-
-```sh
-fluxctl list-controllers --all-namespaces
+fluxctl -f default list-controllers
 ```
 
-### Local endpoint
+This setting can also be set in an environment variable (`FLUX_FORWARD_NAMESPACE`).
 
-**Beware**: this exposes the Flux API, unauthenticated, over an
-insecure channel. Do not do this _unless_ you are operating Flux
-entirely locally; and arguably, only to try it out.
+If you are not able to use the port forward to connect, you can connect by URL with
+`--url` or `FLUX_URL`:
```

Comment:
This is misleading, since there are no preceding instructions for setting up a NodePort or equivalent. I'm happy to remove those, but we need to be clear that the example given won't just work.

Reply:
Updated it to indicate that you need to setup some way of connecting to it.